maven-parent/pom.xml

```diff
@@ -275,7 +276,7 @@
           <execution>
             <!-- Copy OpenMRS modules -->
             <id>Copy OpenMRS modules</id>
-            <phase>prepare-package</phase>
+            <phase>package</phase>
```

Comment:
Why the change in the phase?

Reply:
Probably by mistake. Have removed it now